spec/rails_helper.rb

```diff
@@ -80,6 +80,14 @@
       ActiveJob::Base.queue_adapter = adapter
     end
   end
+
+  WebMock.disable_net_connect!(allow_localhost: true)
+
+  config.around(:each, :vcr) do |example|
```

Comment:
The significance of the `:vcr` condition on this `around()` block wasn't immediately clear to me – I didn't realise that it only applied only to VCR examples, rather than everything. Which made me question why we couldn't just `enable_net_connect!` globally.

A small comment might be helpful (to remind me next time I see this config...):

```suggestion
  # Allow HTTP requests when VCR is used
  config.around(:each, :vcr) do |example|
```